### PR TITLE
test(vertical-divider): migrate cypress tests to playwright

### PR DESCRIPTION
**Pull request by tomdavies73**
### Proposed behaviour

The VerticalDivider component should have in-browser tests using the Playwright framework

### Current behaviour

VerticalDivider currently has Cypress tests instead of Playwright tests

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Playwright automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Related docs have been updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

**Review by edleeks87 — APPROVED** [review]

File: src/components/vertical-divider/vertical-divider.pw.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        page,
+      }) => {
+        const props = { [propName]: pixelVal };
+
+        await mount(<VerticalDividerComponent {...props} />);
+
+        await expect(verticalDividerComponent(page)).toHaveCSS(
+          "height",
+          pixelCssString
+        );
```
Comment (stephenogorman) [inline src/components/vertical-divider/vertical-divider.pw.tsx:48]: Same as previous comment.

File: src/components/vertical-divider/components.test-pw.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import VerticalDivider from "./vertical-divider.component";
+
+type TintRange =
```
Comment (DipperTheDan) [inline src/components/vertical-divider/components.test-pw.tsx:4]: Perfection. *Chef's Kiss*

**Review by DipperTheDan — DISMISSED** [review]

File: src/components/vertical-divider/components.test-pw.tsx (hunk 2 above)
Comment (tomdavies73) [reply src/components/vertical-divider/components.test-pw.tsx:4]: Had to jinx it :`(

**Review by DipperTheDan — APPROVED** [review]

**Pull request merged**